src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4448,26 +4473,48 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     </MSBuild>
 
+    <!-- Projects opting in for 1-level transitive content copying will have to replace OutputItemType metadata from ProjectReference items like so:
+        "Content" to "_OutputItemTypeContentItems"
+        "None" to "_OutputItemTypeNoneItems"
+        "EmbeddedResource" to "_OutputItemTypeEmbeddedResourceItems"
```

Comment:
I don't know of a way to do so offhand but I'd like to avoid this if possible. Will think on it some more.

Reply:
I'm thinking about a different way as well. I realized this is a breaking change for all the things reading the Content item. For example, RAR also probes any potential DLLs in `Content` and `None`, because why not.

Reply:
@rainersigwald I think I fixed it in https://github.com/microsoft/msbuild/pull/4865/commits/152169f04ba13bec681efb6d0a9b4645a2fe686e
Apparently the right magically added metadata is not `DefiningProjectFullPath`, but `MSBuildSourceProjectFile`